### PR TITLE
fix text orientation of winds for CJK

### PR DESCRIPTION
**Pull request by ef81sp**
## TL; DR
- When CJK (Chinese, Japanese, Korean) is displayed in vertical mode, the characters are upright.
- Players' winds are also displayed upright. It is unexpected.
- Fixed them to be displayed sideways.

## Problem
When CJK (Chinese, Japanese, Korean) is displayed in vertical mode, the characters are upright.
Therefore, Shimocha and Kamicha's wind are displayed upright unexpectedly.

This image is in Japanese(東南西北), but Chinese and Korean have the same problem.
image

## Solution
[`text-orientation: sideways`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-orientation) makes CJK in sideways in vertical mode.
image

This change does not affect English.
image

### CODE REVIEW

**Comment by ef81sp** [comment]
I always use this tool to study Mahjong. Thank you.

**Comment by killerducky** [comment]
Nice! I noticed this issue but didn't get around to researching how to fix it. Thanks!
